main.py

```diff
@@ -62,7 +67,7 @@ class DumpFormat(Enum):
 def dump_syntax_tree(
     code: str = Field(description = "The code you need"),
     language: str = Field(description = "The language of the code"),
-    format: DumpFormat = Field(description = "Code dump format. Available values: pattern, ast, cst", default = "cst"),
+    format: DumpFormat = Field(description = "Code dump format. Available values: pattern, ast, cst", default = DumpFormat.CST),
```

Comment:
IIRC using default = DumpFormat.CST will break some MCP clients for some reason.